src/core/handlers/GraphQLHandler.ts

```diff
@@ -33,6 +33,7 @@ export interface GraphQLHandlerInfo extends RequestHandlerDefaultInfo {
 
 export type GraphQLRequestParsedResult = {
   match: Match
+  cookies: Record<string, string>
```

Comment:
unrelated to this directly, however I noticed that we parsed cookies in the http handler in the parse function, but in graphql we only parsed them in the extendedArgs.  we should probably be consistent?

Reply:
Agree. An oversight on my end, most likely. The `cookies` have been added to GraphQL handler recently.

Reply:
@kettanaito I think it should be reasonable to move this outside of this pr, i'll open one with just that change separate from this pr!

Reply:
https://github.com/mswjs/msw/pull/1957

opened this one to land that change separate from any other changes here

Reply:
Amazing! It's been merged.